src/main/java/net/dv8tion/jda/api/interactions/commands/build/OptionData.java

```diff
@@ -596,6 +633,27 @@ public OptionData setRequiredRange(double minValue, double maxValue)
         return this;
     }
 
+    @Nonnull
+    public OptionData setMinLength(int minLength)
+    {
+        if (type != OptionType.STRING)
+            throw new IllegalArgumentException("Can only set min length for options of type STRING");
+        Checks.check(minLength >= 0, "Min length must be greater than or equal to 0");
```

Comment:
would it make sense to add a bound check for the maximum value here as well?

Reply:
I'll add the check. however, according to docs, quote:

> The value of `min_length` must be greater or equal to `0`

this check is correct.